apps/vanilla/src/services/api/register.ts

```diff
@@ -0,0 +1,31 @@
+import { HttpErrorDto } from '@js-camp/core/dtos/httpError.dto';
+import { TokenDto } from '@js-camp/core/dtos/token.dto';
+import { ErrorUserDto } from '@js-camp/core/dtos/user.dto';
+import { HttpErrorMapper } from '@js-camp/core/mappers/httpError.mapper';
+import { TokenMapper } from '@js-camp/core/mappers/token.mapper';
+import { ErrorUserMapper, UserMapper } from '@js-camp/core/mappers/user.mapper';
+import { HttpError } from '@js-camp/core/models/httpError';
+import { Token } from '@js-camp/core/models/token';
+import { ErrorUser, User } from '@js-camp/core/models/user';
+import { AxiosError } from 'axios';
+
+import { appAxios } from '../../axios';
+
+const REGISTER_URL = 'auth/register/';
+
+/** Post user information to register.
```

Comment:
Also, don't forget about formatting
```suggestion
/** Register new user..
```